Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.6
```

Comment:
Is it possible to add more language versions? Since ELFI supports Python starting from version 3.5, it might be useful to have this opportunity.

Reply:
It's possible, but why would you want it?

Reply:
There is no necessity to add another version. I thought that it would be convenient and coherent between documentation and dockerfile

Reply:
I don't think it would be convenient: the installation time and the size of downloads would increase and the user would anyway have to know how to change the python version. If the user for some reason wants to use Python 3.5, it's very easy for him/her to change the Dockerfile accordingly.

The documentation says that ELFI requires Python 3.5+. This PR doesn't change that.

Reply:
I didn't know about installation time and the size. Ok, sorry for asking.